lib/vultr/version.rb

```diff
@@ -1,10 +1,3 @@
 module Vultr
-  module VERSION
-    MAJOR = 0
-    MINOR = 4
-    TINY  = 3
-    PRE   = nil
-
-    STRING = [MAJOR, MINOR, TINY, PRE].compact.join('.')
-  end
+  VERSION = "2.0.0"
```

Comment:
@excid3 Why we skip the 1.0 version and bump to 2.0 directly?

Reply:
I thought since we're implement API v2, maybe we just have the major version match?

Reply:
Changed this to 1.0. I don't think we need to track the same version as their API. 👍